package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.3.4",
     "onchange": "^7.1.0",
+    "prop-types": "^15.8.1",
```

Comment:
I've never used this but seems like a good idea! Is this something you all used in the React class? If so, great! If not, the team might need a demo at some point. Or maybe people who feel comfortable using it can implement it.

Reply:
Yea we used prop-types in the class, as a substitute for Typescript. We used it for sending data types. I'll uninstall the package for now and run it by the team before I start adding types to components.